src/components/RunHeader/RunHeader.js

```diff
@@ -53,6 +53,11 @@ class RunHeader extends Component {
                 </div>
                 <span className="status-label">{reason}</span>
                 <span className="time">{lastTransitionTime}</span>
+                <Rebuild
```

Comment:
RunHeader is reused for TaskRuns, and although it would be useful to have there it doesn't work at the moment so I'd suggest adding this only when `pipelineRun` is set.

Reply:
On it, I'll add
```
      {pipelineRun && (
                  <Rebuild
                    {...this.props}
                    pipelineRun={pipelineRun}
                    runName={runName}
                  />
                )}
```
and retest